build/configs/qemu/tc_64k/defconfig

```diff
@@ -306,7 +308,7 @@ CONFIG_PREALLOC_TIMERS=4
 #
 CONFIG_INIT_ENTRYPOINT=y
 CONFIG_RR_INTERVAL=200
-CONFIG_TASK_NAME_SIZE=32
+CONFIG_TASK_NAME_SIZE=16
```

Comment:
I guess setting to even number makes lot of memory wastage as the code inside OS uses 
include/tinyara/sched.h:588:    char name[CONFIG_TASK_NAME_SIZE + 1]; to  fit into 32 bytes ( 4 words ) 
I request to change it to  15

Reply:
And same may apply to other defconfigs as well , kindly reduce to 31

../build/configs/qemu/tc_16m/defconfig:314:CONFIG_TASK_NAME_SIZE=32
../build/configs/qemu/tc_16m_grpc/defconfig:314:CONFIG_TASK_NAME_SIZE=32
../build/configs/qemu/tc_1m/defconfig:314:CONFIG_TASK_NAME_SIZE=32

Reply:
I think that is a different kind of change. We need to change all of defconfig for that.

Reply:
I've contained changes of tc and configs in this PR because of build break with pthread key changes.
I have no plan to add another changes.